pkgs/development/libraries/freeimage/default.nix

```diff
@@ -51,6 +51,8 @@ stdenv.mkDerivation (finalAttrs: {
       substituteInPlace Makefile.osx --replace "x86_64" "arm64"
     '';
 
+  NIX_LDFLAGS = "-headerpad_max_install_names";
```

Comment:
What does this do? Add comment maybe?

Reply:
It's in commit message. This is introduced by `autoSignDarwinBinariesHook` (attempts to rewrite dylib but fails because not enough space in header) and should be removed in https://github.com/NixOS/nixpkgs/pull/369766. If I get that PR in, this won't be needed.

Reply:
Rebased and removed.